ui/src/index.css

```diff
@@ -1,4 +1,4 @@
-body {
+body, * {
```

Comment:
Pretty sure this `*` selector is redundant; the `body` selector already applies rules to every component below it, since the rules are _cascading_ (CSS). The `*` selector might also have unintended consequences later, as you might have to override sensible defaults on some components.